catalog/acm/config-mgmt-csr.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeatureMembership
+metadata: # kpt-merge: platform-namespace/feature-membership-name
+  name: feature-membership-name # kpt-set: gke-acm-config-mgmt-csr-${cluster-name}
+  namespace: platform-namespace # kpt-set: ${platform-namespace}
+  annotations:
+    cnrm.cloud.google.com/blueprint: cnrm/acm/v0.0.1
+spec:
+  projectRef:
+    external: platform-project-id # kpt-set: ${platform-project-id}
+  location: global
+  membershipRef:
+    name: membership-name # kpt-set: acm-mem-${cluster-name}
+  featureRef:
+    name: feature-name # kpt-set: gke-acm-config-mgmt-${cluster-name}
+  configmanagement:
+    version: 1.9.0
+    configSync:
+      sourceFormat: unstructured # kpt-set: ${sourceFormat}
+      git:
+        syncRepo: https://source.developers.google.com/p/{project_id}/r/{repo} # kpt-set: ${syncRepo}
+        syncBranch: main # kpt-set: ${syncBranch}
+        policyDir: config/root # kpt-set: ${policyDir}
+        syncWaitSecs: "10" # kpt-set: ${syncWaitSecs}
+        syncRev: HEAD # kpt-set: ${syncRev}
```

Comment:
I think we can reduce setters in the blueprint (like these which I dont think regular users would want to change and they can always add setters if they want to)

Reply:
I did not include these in the setters.yaml but left the them here to make is easier to make these variables in the future. I guess its easy for anyone to add them later, if needed. I can remove but is there a harm in leaving them in?

Reply:
The direction we have been planning to go is to reduce usage of setters /cc @morgante. Users can create them on forks easily using the [create-setters fn](https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/create-setters) imperatively.

Reply:
Reduced the number of setters in the package